building/README.md

```diff
@@ -338,6 +340,21 @@ cd phoenix-rtos-project
 (cd phoenix-rtos-build/toolchain/ && ./build-toolchain.sh sparc-phoenix ~/toolchains/sparc-phoenix)
 ```
 
+<details>
+<summary> Errors and warnings during toolchain compilation </summary>
+&nbsp;
+
+If you have encountered some issue during the toolchain build - you probably interrupted a build before or the files in
+the `toolchains` directory are broken for some reason. Removing a directory for a specific architecture
+(arm-phoenix/i386-pc-phoenix/riscv64-phoenix/sparc-phoenix) and launching a build once again should help.
+
+Sometimes while compiling the toolchain some warnings can appear. That behavior is intended in most cases, e.g.:
```

Comment:
It's not intended to have warnings - it's not resolved by us yet I suppose. If we paste some screenshots it should be done according to the screenshots properties, buut I would suggest remove this one - we don't want to brag that we have warnings :smiley: We can just leave the note like `Even during the correct compilation process there may be some unresolved warnings`